client/command.go

```diff
@@ -54,13 +54,13 @@ type VisibilityParams struct {
 	Logf           func(string, ...interface{})
 	Errorf         func(string, ...interface{})
 	SummaryMetrics func(map[string]uint64)
-	InvokeMetrics  func(map[string]string)
+	InvokeMetrics  func(string, string)
 }
 
 var logf = func(string, ...interface{}) {}
 var errorf = func(string, ...interface{}) {}
 var daemonReportMetrics = func(map[string]uint64) {}
-var clientInvokeMetrics = func(map[string]string) {}
+var clientInvokeMetrics = func(string, string) {}
```

Comment:
perhaps we should do a basic struct to capture the tags?
something like
```
struct InvocationMetricTags type {
  ResponseType string
  MethodName string
  ExtraTags map[string]string
}
```

or something along those lines.